tests/modules/programs/zellij/default.nix

```diff
@@ -0,0 +1,9 @@
+{
+  zellij-bash = ./bash.nix;
+  zellij-no-bash = ./no-bash.nix;
+  zellij-no-shell = ./no-shell.nix;
```

Comment:
Where's `no-shell.nix`?

Reply:
Good catch with the `no-shell`. I referenced another package's tests when creating these ones.

I can merge these into two tests: one where all shells are enabled, and another where they are all disabled.

Reply:
Tests cases have been reduced to just two, now.